### PR TITLE
zh-cn(update): sync about notification

### PR DESCRIPTION
**Pull request by fwqaaq**
Part of #3893

### CODE REVIEW

File: files/zh-cn/web/api/notification/index.md [hunk 1]
```diff
@@ -1,152 +1,144 @@
...
-title: notification
-slug: Web/API/notification
+title: Notification
+slug: Web/API/Notification
 ---
 
 {{APIRef("Web Notifications")}}{{AvailableInWorkers}}{{securecontext_header}}
 
-[Notifications API](/zh-CN/docs/Web/API/Notifications_API) 的**通知**接口用于向用户配置和显示桌面通知。
+[Notifications API](/zh-CN/docs/Web/API/Notifications_API) 的接口 `Notification` 用于想用户配置和显示桌面通知。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/api/notification/index.md:8]:
```suggestion
[Notifications API](/zh-CN/docs/Web/API/Notifications_API) 的接口 `Notification` 用于向用户配置和显示桌面通知。
```

File: files/zh-cn/web/api/notification/index.md [hunk 2]
```diff
@@ -1,152 +1,144 @@
...
+- {{domxref("Notification.body")}} {{ReadOnlyInline}}
+  - : 在构造函数的 `options` 参数中指定的通知的正文字符串。
+- {{domxref("Notification.data")}} {{ReadOnlyInline}}
+  - : 返回一个通知数据的结构化克隆。
+- {{domxref("Notification.dir")}} {{ReadOnlyInline}}
+  - : 在构造函数的 `options` 参数中指定的通知的文本方向。
+- {{domxref("Notification.lang")}} {{ReadOnlyInline}}
+  - : 在构造函数的 `options` 参数中指定的通知的语言。
+- {{domxref("Notification.tag")}} {{ReadOnlyInline}}
+  - : 在构造函数的 `options` 参数中指定的通知的 ID（如果有的化）。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/api/notification/index.md:51]:
```suggestion
  - : 在构造函数的 `options` 参数中指定的通知的 ID（如果有的话）。
```

**Comment by yin1999** [comment]
> ### Description
> fix: #3893

Please use `Part of #3893`, or the related issue will be closed when I merge this PR.

File: files/zh-cn/web/api/notification/index.md [hunk 3]
```diff
@@ -1,152 +1,144 @@
...
+- {{domxref("Notification.icon")}} {{ReadOnlyInline}}
+  - : 在构造函数的 `options` 参数中用作指定通知图标的图像 URL。
+- {{domxref("Notification.image")}} {{ReadOnlyInline}} {{Experimental_Inline}}
+  - : 在构造函数的 `options` 参数中用作指定通知的一部分显示的图像 URL。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/api/notification/index.md:55]:
```suggestion
  - : 作为通知的一部分显示的图像 URL，在构造函数的选项参数中指定。
```

**Comment by yin1999** [comment]
To avoid too much noise, could you create a **draft** PR, and convert it to ready for review until you have done all the job?

**Comment by Jack-Zhang-1314** [comment]
> > ### Description
> > fix: #3893
> 
> Please use `Part of #3893`, or the related issue will be closed when I merge this PR.

I should write `Part of #3893`,  rather than `fix: #3893` ?

**Comment by Jack-Zhang-1314** [comment]
> To avoid too much noise, could you create a **draft** PR, and convert it to ready for review until you have done all the job?

yeah, I was checking it

**Comment by yin1999** [comment]
> I should write `Part of #3893`, rather than `fix: #3893` ?

Yes, please.

**Comment by Jack-Zhang-1314** [comment]
> > I should write `Part of #3893`, rather than `fix: #3893` ?
> 
> Yes, please.

why? I don't understand

**Comment by yin1999** [comment]
image

Have you remove all the `page` macro metioned in that issue

**Comment by Jack-Zhang-1314** [comment]
So the GitHub specifies these keywords, for example (`fix`, `part of`) ?

**Comment by Jack-Zhang-1314** [comment]
If I'm using `fix`, then it will close; if I'm using `part of`, then it won't close

**Comment by yin1999** [comment]
> So the GitHub specifies these keywords, for example (`fix`, `part of`) ?

`fix` is used to link with an issue, and when the pr is merged, the linked issue will be closed. See also: <https://docs.github.com/cn/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword>

**Comment by Jack-Zhang-1314** [comment]
> > So the GitHub specifies these keywords, for example (`fix`, `part of`) ?
> 
> `fix` is used to link with an issue, and when the pr is merged, the linked issue will be closed. See also: https://docs.github.com/cn/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

thank you

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Comment by Jack-Zhang-1314** [comment]
image
* [ ]  Part of [#3893]
* github 中这样的 ui 只能使用 markdown 的 to-do 形式才能展现？

File: files/zh-cn/web/api/notification/index.md [hunk 4]
```diff
@@ -1,152 +1,144 @@
...
 
-{{page("/en-US/docs/Web/API/EventTarget","Methods")}}
+- {{domxref("Notification.click_event", "click")}}
+  - : 当用于点击通知时调用。
+- {{domxref("Notification.close_event", "close")}}
+  - : 当用于关闭通知时调用。
+- {{domxref("Notification.error_event", "error")}}
+  - : 当通知发生错误时调用。
+- {{domxref("Notification.show_event", "show")}}
+  - : 当显示通知时调用。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/index.md:92]: 这里翻译有问题，还是请认真核对英文原文

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/api/notification/index.md (hunk 4 above)
Comment (Jack-Zhang-1314) [reply files/zh-cn/web/api/notification/index.md:92]: **用户**，这里打错了

File: files/zh-cn/web/api/notification/index.md [hunk 5]
```diff
@@ -1,152 +1,144 @@
...
 - {{domxref("Notification.close()")}}
-  - : 用于关闭通知。
+  - : 以编程方式关闭通知实例。
 
-`Notification` 对象继承自 {{domxref("EventTarget")}} 接口。
+## 事件
 
-{{page("/en-US/docs/Web/API/EventTarget","Methods")}}
+- {{domxref("Notification.click_event", "click")}}
+  - : 当用于点击通知时调用。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/api/notification/index.md:86]:
```suggestion
  - : 当用户点击通知时调用。
```

File: files/zh-cn/web/api/notification/index.md [hunk 6]
```diff
@@ -1,152 +1,144 @@
...
+- {{domxref("Notification.close_event", "close")}}
+  - : 当用于关闭通知时调用。
```
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/api/notification/index.md:88]:
```suggestion
  - : 当用户关闭通知时调用。
```

File: files/zh-cn/web/api/notification/index.md (hunk 4 above)
Comment (Jack-Zhang-1314) [inline files/zh-cn/web/api/notification/index.md:92]:
```suggestion
  - : 当通知显示时调用。
```

File: files/zh-cn/web/api/notification/index.md [hunk 7]
```diff
@@ -1,152 +1,144 @@
...
 
-{{page("/en-US/docs/Web/API/EventTarget","Methods")}}
+- {{domxref("Notification.click_event", "click")}}
+  - : 当用户点击通知时调用。
+- {{domxref("Notification.close_event", "close")}}
+  - : 当用户关闭通知时调用。
+- {{domxref("Notification.error_event", "error")}}
+  - : 当通知发生错误时调用。
+- {{domxref("Notification.show_event", "show")}}
+  - : 当通知显示时调用。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/index.md:92]: 注意，事件不是被调用，而是被触发：

```suggestion
  - : 当用户点击通知时触发。
- {{domxref("Notification.close_event", "close")}}
  - : 当用户关闭通知时触发。
- {{domxref("Notification.error_event", "error")}}
  - : 当通知发生错误时触发。
- {{domxref("Notification.show_event", "show")}}
  - : 当通知显示时触发。
```

File: files/zh-cn/web/api/notification/index.md [hunk 8]
```diff
@@ -1,152 +1,144 @@
...
+  // At last, if the user has denied notifications, and you
+  // want to be respectful there is no need to bother them anymore.
 }
 ```
 
-### See the live result
+在这里，我们不再展示一个在线实例，因为 Chrome 和 Firefox 不再允许从跨域的 {{htmlelement("iframe")}} 请求通知权限，其他浏览器会陆续跟进。要查看实际示例，请查看我们的[待办事项示例](https://github.com/mdn/dom-examples/tree/main/to-do-notifications)（参见[在线运行的应用程序](https://mdn.github.io/dom-examples/to-do-notifications/)）。
 
-{{ EmbedLiveSample('Example', '100%', 30) }}
+> **备注：** 在上面的示例中，我们生成了响应用户手势的的通知（点击按钮）。这是最好的实践——你不应该向用户发送它们不同意的通知——而且未来浏览器将明确的禁止未响应用户手势发出的通知。例如， Firefox 已经从版本 72 开始就这么做了。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/index.md:132]:
```suggestion
> **备注：** 在上面的示例中，我们生成了响应用户手势的的通知（点击按钮）。这是最佳实践——你不应该向用户发送它们不同意的通知——而且未来浏览器将明确的禁止未响应用户手势发出的通知。例如， Firefox 已经从版本 72 开始就这么做了。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Review by yin1999 — APPROVED** [review]

File: files/zh-cn/web/api/notification/index.md (hunk 7 above)
Comment (Jack-Zhang-1314) [reply files/zh-cn/web/api/notification/index.md]: 👌

**Pull request merged**